README.md

```diff
@@ -256,15 +223,17 @@ Promise.delegate((resolve, reject, resolveWithPresence) => {
 For example,
 
 ```js
-const executor = async (resolve, reject, resolveWithPresence) => {
+const delegatedExecutor = async (resolve, reject, resolveWithPresence) => {
   // Do something that may need a delay to complete.
   const { err, presenceHandler, other } = await determineResolution();
   if (presenceHandler) {
-    // presence is a freshly-created Object.create(null) whose handler
+    // presence is marked as an identity whose handler
     // is presenceHandler.  The targetP below will be resolved to this
     // presence.
-    const presence = resolveWithPresence(presenceHandler);
-    presence.toString = () => 'My Special Presence';
+    const presence = {
+      toString: () => 'My Special Presence',
+    };
+    return resolveWithPresence(presence, presenceHandler);
```

Comment:
As we discussed, we must not enable attack. I think at least this PR should preserve the defensive freshness until we safely know how to avoid it. Please let's leave any such controversial change to #26 , where it will be a central topic for discussion.